client/wordpress-com.js

```diff
@@ -193,6 +193,16 @@ sections.push( {
 	secondary: true
 } );
 
+if ( config.isEnabled( 'account-recovery' ) ) {
+	sections.push( {
+		name: 'account-recovery',
+		paths: [ '/account-recovery' ],
+		module: 'account-recovery',
+		secondary: false,
+		enableLoggedOut: true,
```

Comment:
I think we'll need to add the SSR ready headers then to the top of the related files:
```
/** @ssr-ready **/
```
See: https://github.com/Automattic/wp-calypso/blob/master/docs/server-side-rendering.md anyone from hyperion should also know more